util/index.js

```diff
@@ -14,8 +14,12 @@ const memoize = (fn) => {
     if (type in cache) {
       return cache[type]
     } else {
-      const result = fn(type, customMock, schema, deepLevel)
```

Comment:
It is important to put an entry into the cache before computing mock properties: otherwise if there is a cycle in schema types `memoizedField` will be called recursively until `deepLevel` maxes out.